impl/src/main/java/io/opencensus/impl/metrics/MetricsComponentImpl.java

```diff
@@ -18,7 +18,7 @@
 
 import io.opencensus.implcore.common.MillisClock;
 import io.opencensus.implcore.metrics.MetricsComponentImplBase;
-import io.opencensus.metrics.MetricsComponent;
+import io.opencensus.temporary.metrics.MetricsComponent;
```

Comment:
I would prefer the namespace to be:
import io.opencensus.**implcore**.temporary.metrics.MetricsComponent;

Reply:
Done